src/content/libraries/async/async-await.md

```diff
@@ -16,6 +16,11 @@ To get the most out of this tutorial, you should have the following:
 
 * Knowledge of [basic Dart syntax](/language).
 * Some experience writing asynchronous code in another language.
+* The [`discarded_futures`][] and [`unawaited_futures`][] lints enabled.
+  As with all lints, these lints aren't enabled by default.
```

Comment:
I think this line can be removed since the default, or expected behavior, is that it's not enabled! Other than that, lgtm!